docs/pages/react/solana-wallets/sponsor-solana-gas.mdx

```diff
@@ -0,0 +1,83 @@
+---
+title: Gas sponsorship
+description: How to sponsor gas on Solana
+text: How to sponsor gas on Solana
+link: /react/solana-wallets/sponsoring-gas
+slug: wallets/react/solana-wallets/sponsoring-gas
+---
+
+Account Kit supports fees and rent sponsorship for Solana wallets, enabling users to transact without holding SOL — removing a major barrier to entry for new users of your app.
+
+You can sponsor gas either globally (for all transactions) or on a per-transaction basis.
```

Comment:
I think Solana might not use the term "gas" and instead call it "transaction fees" and "rent." But I could be wrong.